libbeat/docs/upgrading.asciidoc

```diff
@@ -37,6 +37,10 @@ We recommend that you fully upgrade {es} and {kib} to version 8.0
 before upgrading {beats}. The {beats} version must be lower than or equal to the
 {es} version. {beats} cannot send data to older versions of {es}.
 
+Users of the Uptime app should make sure to add `synthetics-*` to their "Index Pattern"
+in the Uptime Kibana app's settings page. Heartbeat 8.0 will prefer the synthetics datastream
+by default, rather than the old `heartbeat-*` indices.
```

Comment:
```suggestion
Users of the Uptime app should make sure to add `synthetics-*` and `heartbeat-8*` to their "Index Patterns"
in the Uptime Kibana app's settings page. Although, Heartbeat 8.0 will default to writing to the synthetics datastream rather than the old `heartbeat-*` indices, there could still be documents indexed into `heartbeat-8*`.
```

Reply:
Is the above accurate? I imagine some people would still stick to the old naming pattern. Should we remove `heartbeat-8*` altogether?